README.md

```diff
@@ -0,0 +1,20 @@
+# [CollabCamp website](http://collab.camp/)
+
+using [Jekyll](http://jekyllrb.com/)
+
+CollabCamp events bring together social entrepreneurs, sharers, hackers, government officials and citizens to collaboratively work on projects which benefit the collaborative economy (sharing). CollabCamp events are free to attend. We celebrate collaboration and sharing. Let's get it done! 
+
+### Editing this site locally
+
+To clone and run a server locally for testing:
+
+    git clone https://github.com/collabcamp/collabcamp.github.io.git
+    cd collabcamp.github.io
+    sudo npm install jekyll
```

Comment:
I'm a bit puzzled why it is on npm - since it's a Ruby package. The description explains that the jekyll gem still needs to be installed for it to work. Which means that you don't need the javascript package at all. I'd think `npm install jekyll` doesn't work when the gem isn't installed.

Anyway - what about following the [official install instructions](http://jekyllrb.com/docs/installation/), optionally adding a note about `sudo`?